nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-policy-management.js

```diff
@@ -759,18 +759,23 @@ nf.PolicyManagement = (function () {
             });
 
             // build the mark up
-            return $('<span>Showing effective policy inherited from Process Group </span>').append($('<span class="link"></span>').text(processGroupName).on('click', function () {
-                // close the shell
-                $('#shell-close-button').click();
-
-                // load the correct group and unselect everything if necessary
-                nf.CanvasUtils.enterGroup(processGroupId).done(function () {
-                    nf.CanvasUtils.getSelection().classed('selected', false);
-
-                    // inform Angular app that values have changed
-                    nf.ng.Bridge.digest();
-                });
-            })).append('<span>.</span>');
+            return $('<span style="position: relative;top: -2px;">Showing effective policy inherited from Process Group </span>').append(
+                $('<span class="link ellipsis" style="max-width: 200px; position: relative; top: 3px;"></span>')
```

Comment:
I'm pretty sure that the div's (`policy-message, new-policy-message, override-policy-message, and add-local-admin-message`) already align. The only issue was the content of `policy-message`. Specifically, the styles in the ellipsis class causes the surrounding text to shift downwards. Setting the link to `vertical-align: middle` should address this.